ui/src/app/login/components/utils.ts

```diff
@@ -153,7 +154,7 @@ export const pkceCallback = async (queryParams: string, oidcConfig: AuthSettings
         throw new PKCELoginError('No token in response');
     }
 
-    document.cookie = `argocd.token=${result.id_token}; path=/`;
+    document.cookie = `argocd.token=${result.id_token}; path=/${requests.toAbsURL('').replace(/^\/|\/$/g, '')}`;
```

Comment:
Can you add a comment to explain the regex replace?

Reply:
I added what the regex pattern does as well as described the issue I discovered that it solves.